### PR TITLE
#160370620 Make multiple requests

### PR DESCRIPTION
**Pull request by seunzone**
#### What does this PR do?
This PR contains details for creating multiple requests by the user

#### Description of Task to be completed?
- A user should be able to see extra input fields when they click on multi-request
- A user should see new fields when they click on add another field
- Users should be able to post all request made to the the API
- Users should be able to delete input fields whilst making requests

#### How should this be manually tested?
- Clone this repo - `git clone https://github.com/andela/travel_tool_front.git`
- Switch to the repo directory - `cd travel_tool_front`
- Check out this branch - `git checkout ft-multitrip-modal-travel-request-160370620`
- Install dependencies and start the server  - `make start`
- Set up the server by following this [guide](https://github.com/andela/travel_tool_back/pull/46)
- Navigate to `localhost:3000` in a browser
- Create a new travel request
- From the Modal that appears, select `Multi City Trip`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#[160370620](https://www.pivotaltracker.com/story/show/160370620)

#### Screenshots (if appropriate)
screen shot 2018-09-15 at 2 58 20 pm

#### Questions:

### CODE REVIEW

**Comment by amarachukwu-agbo** [comment]
Functionality works as expected

File: src/components/Table/Table.scss [hunk 1]
```diff
@@ -79,7 +79,7 @@ td.pl-sm-100 {
     background-color: #f8f8f8;
     width: 96.3%;
     position: relative;
-    overflow: scroll;
+    // overflow: scroll;
```
Comment (daddychukz) [inline src/components/Table/Table.scss]: take this out

Comment (samuelkubai) [reply src/components/Table/Table.scss]: @daddychukz can this be done after the demo?

Is everything else working as expected?

Comment (daddychukz) [reply src/components/Table/Table.scss]: Yes. This PR has been tested and working fine now

**Pull request merged**